deploy/kubernetes/manifests/controller.yaml

```diff
@@ -20,20 +20,18 @@ spec:
       containers:
         - name: csi-external-attacher
           imagePullPolicy: Always
-          image: gcr.io/gke-release/csi-attacher:v0.4.1-gke.0
+          image: gcr.io/gke-release/csi-attacher:v2.2.0-gke.0
```

Comment:
We shouldn't need csi-attacher anymore. We can set CSIDriver.AttachRequired = false

Reply:
That is true, we have only create volume and node publish calls. Thanks for pointing it out.